pkg/monitoring/domainstats/prometheus/prometheus.go

```diff
@@ -301,6 +301,16 @@ func (metrics *vmiMetrics) updateVcpu(vcpuStats []stats.DomainStatsVcpu) {
 				[]string{stringVcpuIdx},
 			)
 		}
+		if vcpu.DelaySet {
+			metrics.pushCustomMetric(
+				"kubevirt_vmi_vcpu_delay_seconds_total",
+				"Amount of time spent by each vcpu waiting in the queue instead of running.",
+				prometheus.CounterValue,
+				float64(vcpu.Delay)/float64(1000000000),
```

Comment:
[PiotrProkop](https://github.com/PiotrProkop) can you please explain why you divided by 1000000000, in order to get to seconds?
I'm not sure what is the unit for domstats vcpu.delay, it's not mentioned in https://www.libvirt.org/manpages/virsh.html#domstats.

Reply:
For vcpu.time and cpu.wait it's microseconds though (1000000).

Reply:
Yeah, I guess you're right I have assumed that all domstats metrics are reported in nanoseconds. Let me make a fix PR for that.

Reply:
No need for a PR. I'm going to fix this for other metrics (`kubevirt_vmi_vcpu_seconds_total` and `kubevirt_vmi_vcpu_wait_seconds_total`), so I can fix `kubevirt_vmi_vcpu_delay_seconds_total` as well.
My only question is, do you know if vcpu.Delay is also in microseconds? It's not mentioned in https://www.libvirt.org/manpages/virsh.html#domstats.

Reply:
This metric is in micro seconds, it's taken from `/proc/stat` from within VM :) https://man7.org/linux/man-pages/man5/proc.5.html

Reply:
Thanks. I'll submit a PR asap.

Reply:
Can you please point to the part in this file, where microseconds is mentioned for the vcpu.Delay?

Reply:
```
The amount of time, measured in units of USER_HZ
                     (1/100ths of a second on most architectures, use
                     sysconf(_SC_CLK_TCK) to obtain the right value),
                     that the system ("cpu" line) or the specific CPU
                     ("cpuN" line) spent in various states:
```